### DISCUSSION TITLE
Cyberduck Windows not prompting to import WinSCP

### DISCUSSION THREAD

**Discussion by IrealiTY** [iterate-ch/cyberduck#16278] [Q&A]
Hi :) I'm trying to port my WinSCP bookmarks to Cyberduck to use them on MacOS
But I can't get the App to prompt me for any import options. WinSCP is set to the .INI config options, as noted in the Cyberduck documentation. (I tried both the default and custom location options)

**Comment by dkocher** [comment]
The expected location for the bookmarks from `WinSCP` on Windows is `%AppData%\WinSCP.ini`.

**Comment by IrealiTY** [comment]
Thanks for the reply but it doesn't seem to prompt it either, I'll try reinstalling Cyberduck later to test it.

**Comment by benyrfg** [comment]
Hi, I have the same issue. I have the config file from WInSCP here: C:\Users\{name}\AppData\Roaming\WinSCP.ini but Cyberduck does not prompting for import. 

I tried reinstall Cyberduck, but it doesn't help
